mllib/src/main/scala/org/apache/spark/mllib/linalg/Vectors.scala

```diff
@@ -424,6 +424,28 @@ object Vectors {
     }
   }
 
+  /**
+    * Returns the mean centered vector
```

Comment:
@shahidki31 . Could you fix the scalastyle first in order to test Jenkins tests successfully?

Reply:
Thank you @dongjoon-hyun . I have updated the code.